.env

```diff
@@ -22,6 +22,14 @@ OKP4_LICENCE_ASSETS_URL='https://creativecommons.org/licenses/by-sa/4.0'
 OKP4_ARBITRAGE_URL='https://www.arbitrage.org'
 OKP4_ARBITRAGE_RULES_URL='https://www.arbitrage.org/arbitrage/#REGLEMENT_ARBITRAGE'
 OKP4_COOKIE_POLICY_URL='https://okp4.network/cookie-policy'
+OKP4_WHITEPAPER_URL='https://docs.okp4.network/whitepaper/abstract'
+OKP4_TECHNICAL_DOCS_URL='https://docs.okp4.network/tech-documentation/ecosystem-overview'
+OKP4_TUTORIAL_VIDEOS_URL='#'
+OKP4_GITHUB_OKP4D_URL='https://github.com/okp4/okp4d'
+OKP4_GITHUB_CONTRACTS_URL='https://github.com/okp4/contracts'
+OKP4_GITHUB_AWESOME_URL='https://github.com/okp4/awesome'
+OKP4_DISCORD_CHANNEL_URL='https://discord.com/channels/946759919678406696/1095341768691695687'
+OKP4_DISCORD_TICKET_CHANNEL_URL='https://discord.com/channels/946759919678406696/1019918731364802621'
```

Comment:
As I said in `config.type.ts` file, do not repeat variables already declared in the file, you can directly use them.